# Msg containing different external stimuli and if they are active

string stimulus
bool status
float32 value